atomicfu-common/src/main/kotlin/kotlinx/atomicfu/Trace.kt

```diff
@@ -0,0 +1,7 @@
+package kotlinx.atomicfu
+
+public expect class Trace(size: Int) {
+    public fun append(text: String)
```

Comment:
`fun append` shall be `@PublishedApi internal`, because bytecode processor only support code patterns that result from its usage from inside of `operator fun invoke`. Direct invocations of `append` in client code are not supported.

Reply:
Note, you shall also use `@JvmName("append")` in additional to marking it as `internal`, so that there is a stable JVM name you can match in bytecode processor.